src/main/java/org/embulk/filter/calcite/PageConverter.java

```diff
@@ -1,100 +1,87 @@
 package org.embulk.filter.calcite;
 
+import java.math.BigDecimal;
+import java.util.TimeZone;
+
 import org.embulk.spi.Column;
 import org.embulk.spi.ColumnVisitor;
 import org.embulk.spi.PageReader;
 import org.embulk.spi.Schema;
 
-import java.math.BigDecimal;
-import java.util.TimeZone;
-
 /**
  * This class converts Embulk's Page values into Calcite's row types. It refers to
  * org.apache.calcite.adapter.csv.CsvEnumerator.
  */
 public class PageConverter
-        implements ColumnVisitor
-{
+        implements ColumnVisitor {
```

Comment:
We can have this on the same line with `class`. Others as well. It's up to you, though.